io/in.go

```diff
@@ -0,0 +1,29 @@
+package io
+
+import (
+	"os"
+
+	"github.com/googleinterns/terraform-cost-estimation/jsdecode"
+	tfjson "github.com/hashicorp/terraform-json"
+)
+
+// GetPlan receives the input file name to extract the plan structure or return an error.
+// The file is also closed if it was successfully opened.
+func GetPlan(inputName string) (*tfjson.Plan, error) {
+	fin, err := os.Open(inputName)
+	if err != nil {
+		return nil, err
+	}
+
+	plan, err := jsdecode.ExtractPlanStruct(fin)
+	if err != nil {
+		fin.Close()
```

Comment:
Prefer using `defer fin.Close()`  after the file was successfully opened to make sure it is always being closed.

Something like:

```
func GetPlan(inputName string) (*tfjson.Plan, error) {
	f, err := os.Open(inputName)
	if err != nil {
		return nil, err
	}

        defer f.Close()

	plan, err := jsdecode.ExtractPlanStruct(f)
	if err != nil {
		return nil, err
	}

	return plan, nil
}
```